script.js

```diff
@@ -0,0 +1,89 @@
+const minSearch = (firstNum, secondNum, thirdNum, fourthNum) => {
+  if (typeof (firstNum && secondNum && thirdNum && fourthNum) === "number") {
+    // console.log(Math.min(firstNum, secondNum, thirdNum, fourthNum));
+    return Math.min(firstNum, secondNum, thirdNum, fourthNum);
+  } else {
+    // console.log("Wrong number of arguments");
+    return "false";
```

Comment:
```suggestion
    return "Something went, wrong. We cannot find the minimum number";
```

Why would you need `"false"` in a string? 🤔